rules/linux/credential_access_credential_dumping.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2023/02/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/02/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the unshadow utility which is part of John the Ripper,
+a password-cracking tool on the host machine. Malicious actors can use the utility to retrieve
+the combined contents of the '/etc/shadow' and '/etc/password' files.
+Using the combined file generated from the utility, the malicious threat actors can use them as input
+for password-cracking utilities or prepare themselves for reconnaissance attacks by gathering credential information
+of the victim to plan future operations.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "endgame-*"]
```

Comment:
Can you provide a screenshot/event with this being triggered on Endgame?

Reply:
@shashank-elastic this one was marked as resolved without any comments or evidence.

Reply:
Please check the issue originally where all test data were submitted!.